apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -22,6 +31,35 @@ ConversationView.prototype = {
     return this.accessors.carrierHeaderPhoneNumber.text();
   },
 
+  get attachments() {
```

Comment:
Some duplication between NewMessage and Conversation, but these two different views after all and don't care much since this is test code :) But let me know if you want more sophisticated solution to reduce duplication